.github/workflows/deploy-auth.yml

```
@@ -0,0 +1,59 @@
+      - name: 🧪 Build JAR
+        run: ./gradlew clean build -x test
+
+      - name: 🔖 Set Git SHA tag
+        id: vars
+        run: echo "SHORT_SHA=$(echo $GITHUB_SHA | cut -c1-7)" >> $GITHUB_ENV
+
+      - name: 🐳 Docker Build & Push
+        run: |
+          docker build -t ${{ secrets.DOCKER_USERNAME }}/auth-service:${SHORT_SHA} .
+          docker tag ${{ secrets.DOCKER_USERNAME }}/auth-service:${SHORT_SHA} ${{ secrets.DOCKER_USERNAME }}/auth-service:latest
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)